lib/justice_gov_sk/infrastructure.rb

```diff
@@ -0,0 +1,19 @@
+module JusticeGovSk
+  module Infrastructure
+    # TODO implement async?
+
+    def crawl
+      (1..pages).each do |page|
+        url = uri.build(page: page)
```

Comment:
> A URI can be further classified as a locator, a name, or both. The term “Uniform Resource Locator” (URL) refers to the subset of URIs that, in addition to identifying a resource, provide a means of locating the resource by describing its primary access mechanism (e.g., its network “location”).

Podla tohto by sme vsetko mali mat na URL, ale so starou db by to potom kolidovalo. listy by som dal na URL a resource na URI, tak ako to je vlastne teraz v kode vsade.